waving_hands/gamemaster.py

```diff
@@ -3,19 +3,41 @@
 import select
 import socket
 import sys
+import logging
+import logging.config
+
+from waving_hands.elemental import Elemental
+from waving_hands.minion import Minion
+from waving_hands.targetable_client import TargetableClient
+from waving_hands.wizard import Wizard
+from waving_hands import groblenames
+
+log = logging.getLogger(__name__)
 
-from elemental import Elemental
-from minion import Minion
-from targetable_client import TargetableClient
-from wizard import Wizard
-import groblenames
 
 class Gamemaster:
 
-    def __init__(self):
+    def __init__(self,
+        host: str = "localhost",
+        port: int = 12345,
+        pregame: bool = True,
+        customize_wizards: bool = True,
+        players: int = 2):
+        """
+        Start the game, with a given number of parameters
+
+        :param host: IP address or host to serve the game, defaults to localhost
+        :param port: Port to use for the server
+        :param pregame: Start the game with pre-game flair? Defaults to True
+        :param customize_wizards: Allow players to customize their wizards? Defaults to True
+        :param players: Number of players to use for the game. Defaults to 2
+        """
```

Comment:
It's always a bit of an opinionated thing for what should be a positional argument and what should be a keyword argument like host and port above. I made them all keyword to preserve the old functionality, in case we want to invoke Gamemaster the way it used to work with just `Gamemaster()`

Reply:
Oh, and doc-strings above are compatible with [Sphinx](https://www.sphinx-doc.org/en/master/), in case you wanted to put together a [read-the-docs](https://about.readthedocs.com/?ref=readthedocs.com) page. It's easy, free, and doesn't take long!